client/methods/unsetUserReason.js

```diff
@@ -0,0 +1,6 @@
+Meteor.methods({
+	unsetUserReason(userId) {
+		Meteor.users.update(userId, { $unset: { 'reason' : 1 } });
+		return true;
+	}
+});
```

Comment:
This method shouldn't exist. First, it is not checking for user permission to unset reason, and second you could unset reason in the same method `setUserActiveStatus`.